### ISSUE TITLE
Error when trying to submit Rating

### ISSUE THREAD

**Issue by simonkorz** [coolcrowd/object-service#178]
When trying to submit the following example rating to the worker-service, 

``` json
{
"rating":0,
"ratingId":8,
"experiment":1,
"answerId":2,
"feedback":"",
"constraints":[1]
}
```

the object-service responds with:

```
MessageOrBuilder must have field set: rating
```

No stack trace availiable.

**Comment by LeanderK** [comment]
i am on it!

**Comment by kelunik** [comment]
Fixed via #182
